src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3504,7 +3521,7 @@ export class ResultsViewPageStore {
         ) {
             return this.mutationMapperStoreByGeneWithDriverKey[
                 this.getGeneWithDriverKey(gene)
-            ]
+            ] && !this.mutationsTableColumnVisibilityModified
```

Comment:
I could store `mutationsTableColumnVisibility` in `ResultsViewMutationMapperStore` itself, and just pass a function to modify that local variable. That way modifying the column visibility doesn't create a new mutation mapper store.

Reply:
I just realized that we had a similar problem with persisting the track visibility. Looks like we have store called `MutationMapperUserSelectionStore`. Can you investigate if we can implement the column visibility the same way by using this store instead?

Reply:
Yeah I was able to create a `MutationMapperColumnVisibilityStore` and that fixed these problems. Good call.

Reply:
Thanks! 

Did you try adding the new observable and action to the same `MutationMapperUserSelectionStore` instead? It might be better to store all the user selections in one store. What do you think?

Reply:
Good idea! I didn't even think about that, but it makes sense. I've updated the PR to incorporate your suggestion.